### PR TITLE
Add toString() method to model and Mover.java

### PR DESCRIPTION
**Pull request by duzhen1996**

### CODE REVIEW

**Comment by huafengw** [comment]
Please use `String.format()`

**Comment by duzhen1996** [comment]
I don't think it is necessary to replace string concatenation with String.format. Because string concatenation is better in performance and more readable.
https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java

**Comment by duzhen1996** [comment]
@huafengw

**Comment by huafengw** [comment]
It's not about performance, I think the top voted [answer](https://stackoverflow.com/a/925444) already explained it.

File: smart-common/src/main/java/org/smartdata/model/ActionType.java [hunk 1]
```diff
@@ -61,4 +61,9 @@ public static ActionType fromName(String name) {
     }
     return null;
   }
+
+  @Override
+  public String toString() {
```
Comment (huafengw) [inline smart-common/src/main/java/org/smartdata/model/ActionType.java]: Do we really need to override this?

File: smart-common/src/main/java/org/smartdata/model/CmdletState.java [hunk 2]
```diff
@@ -56,4 +56,9 @@ public static boolean isTerminalState(CmdletState state) {
         || state.equals(DISABLED)
         || state.equals(FAILED);
   }
+
+  @Override
+  public String toString() {
```
Comment (huafengw) [inline smart-common/src/main/java/org/smartdata/model/CmdletState.java]: And also this one. I think enum's toString is good enough.

**Comment by huafengw** [comment]
And at last but not least, please pay attention to code format, I noticed some unneeded blank lines. If you are using intellij, I recommend you to install a plugin called `google-java-format` to format your code.

**Comment by qiyuangong** [comment]
PR closed.

**Pull request closed without merging**